packages/blockchain-info-components/src/Form/SelectInput/index.js

```diff
@@ -21,18 +21,28 @@ class SelectInputContainer extends React.PureComponent {
   /* eslint-enable react/no-did-update-set-state */
 
   handleChange = (item) => {
-    const value = prop('value', item)
+    const value = Array.isArray(item) ? item : prop('value', item)
+
+    if (Array.isArray(value)) {
+      const values = []
+      item.map((it) => values.push(it))
+
+      this.setState({ value: values })
```

Comment:
Instead of iterating and creating a new array just do `this.setState({ value: [...item] })`